base/div.jl

```diff
@@ -30,6 +30,10 @@ julia> div(-5, 2, RoundNearestTiesAway)
 -3
 julia> div(-5, 2, RoundNearestTiesUp)
 -2
+julia> div(4, 3, RoundFromZero)
```

Comment:
Should there be a "!! compat needs Julia 1.8" message somewhere? Perhaps in this docstring for `round`, perhaps elsewhere?

Reply:
Added notes to docs in `RoundFromZero`, `div`, `rem` and `RoundingMode`.